standard/statements.md

```diff
@@ -2018,6 +2054,101 @@ using (ResourceType rN = eN)
 >
 > *end example*
 
+A `using` statement of the form
+
+```csharp
+await using (ResourceType resource = «expression» ) «statement»
+```
+
+corresponds to the expansions shown above with `IAsyncDisposable` instead of `IDisposable`, `DisposeAsync` instead of `Dispose`, and the `Task` returned from `DisposeAsync` is `await`ed. An `await using` of the form:
+
+```csharp
```

Comment:
This duplication feels odd to me. Can we not just amend the previous paragraph to end with "The corresponding expansion is"